PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp

```diff
@@ -739,7 +739,7 @@ BrowserNodeType FileBrowser::getNodeType(HTREEITEM hItem)
 		return browserNodeType_file;
 	}
 	// Root
-	else if (tvItem.lParam != NULL && !reinterpret_cast<SortingData4lParam*>(tvItem.lParam)->_rootPath.empty())
+	else if (tvItem.lParam != 0 && !reinterpret_cast<SortingData4lParam*>(tvItem.lParam)->_rootPath.empty())

```

Comment:
Semantic problem: it could be `nullptr` but not `0`.

Reply:
`LPARAM` is not a pointer but an integer. So it's not right to compare it with or assign a `NULL` to it.
It is a defined as following in [minwindef.h](https://github.com/tpn/winsdk-10/blob/master/Include/10.0.16299.0/shared/minwindef.h#L187):
`typedef LONG_PTR            LPARAM;`